src/ensembl/webpack/webpack.config.dev.js

```diff
@@ -64,6 +64,9 @@ const devConfig = {
       });
     },
 
+    // disable host check only when testing on IE11 remotely
+    disableHostCheck: false,
```

Comment:
Ok, I am confused. False is the default value. Why do we want to add it? So that it becomes more obvious which field to change when working with IE11?